lib/lightgbm/booster.rb

```diff
@@ -241,6 +245,11 @@ def read_int64(ptr)
       ptr.read_array_of_int64(1).first
     end
 
+    def sorted_feature_values(input_hash)
+      @cached_feature_names ||= feature_name
+      input_hash.transform_keys(&:to_s).fetch_values(*@cached_feature_names)
```

Comment:
I'm considering if accepting the input hash with keys as symbol or string is worth it, or if it should expect only strings.

Quick benchmark (10000 iterations):

```
                user     system      total        real
   keys are strings:  0.031401   0.001063   0.032464 (  0.032646)
with transform_keys:  0.108264   0.005040   0.113304 (  0.113892)
```

```suggestion
      input_hash.fetch_values(*@cached_feature_names)
```